src/rosdep2/installers.py

```diff
@@ -278,8 +278,9 @@ def get_depends(self, rosdep_args):
         """
         return []  # Default return empty list
 
-    def resolve(self, rosdep_args_dict):
+    def resolve(self, rosdep, rosdep_args_dict):
```

Comment:
Can you please move this argument to after the `rosdep_args_dict` and add a default value for it so it doesn't unnecessarily break other projects that rely on `resolve`?

Reply:
@russkel thanks for comment, this makes much simpler, fixed the code